### ISSUE TITLE
[YouTube Downloader] ... homebrew path

### ISSUE THREAD

**Issue by russ2204** [raycast/extensions#15768]
### Extension

https://www.raycast.com/vimtor/youtube-downloader

### Raycast Version

1.87.4

### macOS Version

15.1.1

### Description

Hi there
I have an iMac using an intel chip and a macbook air using a silicon chip.  WHen setting up the extension it seems the location for the homebrew file is different depending on the chip!  How do I get around this so I can use the extension on either device ?

### Steps To Reproduce

please advise

### Current Behaviour

as described

**Comment by remorses** [comment]
You can customize the ffmpeg path in the settings

**Comment by russ2204** [comment]
Yes I can see how it’s done but this will affect all devices as it’s updated in the extension. Is there a way of it affecting both devices.
